src/OpenTelemetry.PersistentStorage.Abstractions/README.md

```diff
@@ -9,5 +9,5 @@ persistent storage.
 ## Installation
 
 ```shell
-dotnet add package --prerelease OpenTelemetry.PersistentStorage.Abstractions
+dotnet add package OpenTelemetry.PersistentStorage.Abstractions
```

Comment:
Is this needed? (do we expect anyone to install this package at all?)

Reply:
In most cases NO, but if someone wants to have their own internal implementation then it is an option.

Reply:
I'm confused, if someone wants to have their own implementation, they should normally do this by changing the code rather than using CLI?

Reply:
Removing this as this is not useful for users implementing abstractions